.github/workflows/testing_framework.yaml

```diff
@@ -1,9 +1,13 @@
 name: Testing Framework
 
 on:
+  push:
```

Comment:
That won't work because the containers org doesn't have the right runner for running the workflows. It is running on the containers-mirror/ai-lab-recipes repo after the pushed commit is synced.

Reply:
k removing